examples/custom-component/components/Clock.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
+import { createComponent } from '@toolpad/studio/browser';
+
+export interface ClockProps {
+  msg: string;
+}
+
+function Clock({ msg }: ClockProps) {
+  const [time, setTime] = React.useState(() => new Date().toLocaleTimeString());
```

Comment:
Maybe some whitespace between the state, effect and render makes it more readable?